plants/species/Hordeum_vulgare/Hordeum_vulgare_hid055_about.md

```diff
@@ -0,0 +1,19 @@
+**About *Hordeum vulgare***
+-------------------------
+Barley, a member of the grass family, is a major cereal grain grown in 
+temperate climates globally. It was one of the first cultivated 
+grains; it was domesticated in the Fertile Crescent around 9000 BC, 
+giving it nonshattering spikelets and making it much easier to 
+harvest. Its use then spread throughout Eurasia by 2000 BC. Barley 
+prefers relatively low temperatures and well-drained soil to grow. It 
+is relatively tolerant of drought and soil salinity but is less 
+winter-hardy than wheat or rye.
```

Comment:
This is the only `about.md` file with more description than the rest. Is it expected?